### PR TITLE
Dockerisation & WordPress 4.2.3 & mno-php 0.9.0

### PR DESCRIPTION
**Pull request by ouranos**
- Update to latest version (4.2.3)
- WordPress on docker
- Update mno-php library using composer

### CODE REVIEW

**Comment by ouranos** [comment]
@BrunoChauvet please review :)

File: deploy/ansible/roles/wordpress-install/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  tags:
+    - update
+
+- name: WordPress | Configure Apache virtual host
+  template:
+    src: etc-apache24-confd-wordpress-conf
+    dest: /etc/apache2/sites-available/wordpress.conf
+    owner: www-data
+    group: www-data
+    mode: 0755
```
Comment (BrunoChauvet) [inline deploy/ansible/roles/wordpress-install/tasks/main.yml:76]: We should probably have the `configuration` tag here as it is server hostname dependent

File: deploy/ansible/roles/wordpress-install/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    state: restarted
+
+- name: WordPress | Configuration file config.php
+  template:
+    src: wp-config.php
+    dest: "{{ wordpress_root_path }}/wp-config.php"
+    owner: www-data
+    group: www-data
+    mode: 0500
+  # tags:
```
Comment (BrunoChauvet) [inline deploy/ansible/roles/wordpress-install/tasks/main.yml]: I'd say the configuration tag is required here in case database credentials are changed

**Comment by ouranos** [comment]
done

**Pull request merged**